### ISSUE TITLE
Contig path not continuous in unitig graph

### ISSUE THREAD

**Issue by snurk** [marbl/canu#700]
Dear developers, with Canu 1.6 I ended up assembling a contig that jumps between connectivity components in unitig graph.
Is it OK?

**Comment by skoren** [comment]
That seems suspicious, the contig should have a path in the unitig graph. What do the bed and gfa files report?

**Comment by snurk** [comment]
I have 8+Mb contig: ctg00019896
>tig00019896 len=8183401 reads=62303 covStat=46664.44 gappedBases=no class=contig suggestRepeat=no suggestCircular=no

Grepping cyano.unitigs.bed for 00019896 gives:
```
ctg00019896     0       13648   utg00000337     0       +
ctg00019896     1109    10888   utg00000338     0       +
ctg00019896     2709    87698   utg00000339     0       +
ctg00019896     78514   362498  utg00000340     0       +
ctg00019896     353781  1713955 utg00000341     0       +
ctg00019896     1703515 1829211 utg00000342     0       +
ctg00019896     1821087 2105427 utg00000343     0       +
ctg00019896     2095656 3622858 utg00000344     0       +
ctg00019896     3611765 3816771 utg00000345     0       +
ctg00019896     3806741 5258360 utg00000346     0       +
ctg00019896     5256294 5267230 utg00000347     0       +
ctg00019896     5264273 5486263 utg00000348     0       +
```
It seems that the info is not available for the last 2.7Mb of bases.
At the same time BLAST from within Bandage reports those 2.7Mb as aligned to a single unitig (tig00000350 len=2701318). At the same time utg00000350 is missing from cyano.unitigs.bed file.

In the Bandage it looks like this:
![image](https://user-images.githubusercontent.com/988777/32704389-3cc5f2a8-c815-11e7-88be-cd49a159ec88.png)
The contig of interest is green and mentioned unitig is its disconnected part (double checked gfa file, Bandage graph visualization is correct).

**Comment by brianwalenz** [comment]
This is probably fixed by 9341a0c2d58c8f5267a0d46f98717705eefca98d and e42d54d4f1b1133b8e944b09733806bfe63bc600, but I don't know if that problem existed back this far.
